### ISSUE TITLE
[rfe] Add dark gray themes along with black ones

### ISSUE THREAD

**Issue by gaydenko** [AnySoftKeyboard/AnySoftKeyboard#680]
Hi! At first, thanks for the app!
Probably I'm wrong, I feel last years there is a strong accessibility tendency to prefer dark gray background color rather than absolute black one for dark themes.

**Comment by menny** [comment]
@gaydenko this is an interesting observation. Can you point me to some examples? I'd like to verify colors.

**Comment by gaydenko** [comment]
I guess standard Android colors are examples of dark gray rather black :)

**Comment by drac0300** [comment]
But on the other hand, on Amoled screens which are getting more and more popular on modern smartphones, absolute black allows you to save battery (as the pixels are off)

**Comment by gaydenko** [comment]
@drac0300 There are dark theme in place already :) Sorry, I wasn't precise. I don't mean replacing existing black keyboards but mean to add dark gray also.

**Comment by menny** [comment]
How about this:
![screenshot_20160731-214655-01](https://cloud.githubusercontent.com/assets/481272/17281223/c1044174-5768-11e6-9c50-16a9eb92ae6d.jpeg)

**Comment by gaydenko** [comment]
@menny Yes, this background would be excellent to my taste!
(I use layout variant containing top bar with 123, arrows and language switch)

**Comment by gaydenko** [comment]
Hearty thanks!!
